cni/pkg/config/config.go

```diff
@@ -39,6 +39,8 @@ type InstallConfig struct {
 	CNINetworkConfigFile string
 	// CNI config template string
 	CNINetworkConfig string
+	// Whether to watch and reinstall CNI configuration and binary files
+	CNIFileWatchEnabled bool
```

Comment:
Can we add this new option to the configuration print at line 116?

Reply:
done